plugins/user/cpubyuser

```diff
@@ -39,9 +48,12 @@
 
 . "$MUNIN_LIBDIR/plugins/plugin.sh"
 
-OTHER_FIELD="others"
-[ "$USERS" = "ALL" ] && USERS=$(w --no-header | awk '{ print $1 }' | sort | uniq)
-
+#[ -z "$OTHER_FIELD" ] && OTHER_FIELD="others"
+[ -z "$USERS" ] && USERS=ALL
+if [ "$USERS" = "ALL" ]; then
+	LU="$(ps ax --format uname | tail +2 | sort -u | grep -v -e '^root$')"
```

Comment:
The current documentation says, that only logged in users are accounted, if `ALL` is given. Your approach would change this.

Instead I would suggest to add another configuration variable, that allows to pick the suitable approach (and keeps the current behaviour unchanged).

Reply:
Done.